tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/bottom_bar.py

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import Base
+
+class BottomBar(Base):
+    _remove_locator = (By.ID, 'remove')
+    _done_locator = (By.ID, 'done')
+
+    def tap_remove(self):
+        self.marionette.find_element(*self._remove_locator).tap()
+        
+        from gaiatest.apps.homescreen.regions.confirm_dialog import ConfirmDialog
+        return ConfirmDialog(self.marionette)
+        
+    def tap_done(self):
```

Comment:
A name that reflects more the user's intent can be: `exit_edit_mode()`. What do you think?